src/test/java/org/opentripplanner/apis/gtfs/mapping/RouteRequestMapperTest.java

```diff
@@ -57,45 +56,6 @@ class RouteRequestMapperTest implements PlanTestConstants {
       );
   }
 
-  @Test
-  void parkingFilters() {
-    Map<String, Object> arguments = Map.of(
-      "parking",
-      Map.of(
-        "unpreferredCost",
-        555,
-        "filters",
-        List.of(
-          Map.of(
-            "not",
-            List.of(Map.of("tags", List.of("wheelbender"))),
-            "select",
-            List.of(Map.of("tags", List.of("roof", "locker")))
-          )
-        ),
-        "preferred",
-        List.of(Map.of("select", List.of(Map.of("tags", List.of("a", "b")))))
-      )
-    );
-
-    var env = executionContext(arguments);
-
-    var routeRequest = RouteRequestMapper.toRouteRequest(env, context);
-
-    assertNotNull(routeRequest);
-
-    final VehicleParkingRequest parking = routeRequest.journey().parking();
-    assertEquals(
-      "VehicleParkingFilterRequest{not: [tags=[wheelbender]], select: [tags=[locker, roof]]}",
-      parking.filter().toString()
-    );
-    assertEquals(
-      "VehicleParkingFilterRequest{select: [tags=[a, b]]}",
-      parking.preferred().toString()
-    );
-    assertEquals(555, parking.unpreferredCost());
-  }
-
```

Comment:
Is there a replacement for this test? It seems quite important to me.

Reply:
I think I've removed this by accident, thought it was testing a different class. I'll add this back.

Reply:
Added the tests back